crates/ruma-federation-api/src/query/get_profile_information/v1.rs

```diff
@@ -59,7 +59,11 @@ impl Response {
 }
 
 /// Profile fields to specify in query.
+///
+/// This type can hold an arbitrary string. To check for formats that are not available as a
+/// documented variant here, use its string representation, obtained through `.as_str()`.
 #[derive(Clone, Debug, PartialEq, Eq, StringEnum)]
+#[non_exhaustive]
 pub enum ProfileField {
```

Comment:
This is not an error type so this change doesn't make sense under the given commit message. Otherwise looks good though.